src/frontend/Komponenter/Behandling/Vilkårpanel/Informasjonsrad.tsx

```diff
@@ -6,23 +6,27 @@ import { FC, ReactNode } from 'react';
 import { mapIkon, VilkårInfoIkon } from './VilkårInformasjonKomponenter';
 
 interface Props {
-    ikon: VilkårInfoIkon;
+    ikon?: VilkårInfoIkon;
     label: string;
-    verdi: ReactNode;
+    verdi?: ReactNode;
```

Comment:
Hvorfor trengs dette? `undefined` er en gyldig ReactNode, men hender det at at vi ikke sender med noenting som verdi?

Reply:
Den trengs for tilfeller som dette hvor labelen er verdien 
<img width="677" alt="image" src="https://user-images.githubusercontent.com/46678893/218433872-11137a21-561e-43b9-8da4-37a7ea9ed902.png">

Så slipper man å sende inn `verdi={undefined}`